src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,20 @@
+package duke.task;
+
+import duke.exception.DukeInvalidAddTaskException;
+
+public class Event extends Task {
+    private String eventTime;
+
+    public Event(String description) throws DukeInvalidAddTaskException { //will get problem?
```

Comment:
you can remove the comment here as it seems more like a "note to self" and not communicating with any other developers